CHANGELOG.md

```diff
@@ -3,6 +3,28 @@
 * Support AGP 4.2
```

Comment:
Can you re-gig this to make it more digestable?

The RN CLI changes are intended for anyone to use directly, so I'd be tempted to combine all the PR's into a single entry.

Reply:
Amended changelog as requested